src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -86,7 +87,8 @@ export const resourceOptions = [
         label: 'API Tutorial',
         url: 'https://api.usaspending.gov/docs/intro-tutorial',
         enabled: true,
-        shouldOpenNewTab: true
+        shouldOpenNewTab: true,
+        externalLink: true
```

Comment:
This object should also have `internalDomain: true` so that this modal does not show up for a USAspending domain
<img width="364" alt="Screen Shot 2021-06-22 at 9 25 27 AM" src="https://user-images.githubusercontent.com/7108785/122932897-20fecc80-d33c-11eb-8118-822f34a0bcb9.png">